### PR TITLE
Add multiple authors filtering support

### PR DESCRIPTION
**Pull request by tomasz-biernacki**
For each author, I'd like to generate a page with all his/her posts.
To keep backwards compatibility this would be enabled only if the `author-root-uri` key is not `nil` (in the `config.edn` file). It would work just like `page-root-uri` or `post-root-uri`.

### CODE REVIEW

**Comment by tomasz-biernacki** [comment]
If you think it could be merged, then also the documentation needs to be updated.

File: src/cryogen_core/compiler.clj [hunk 1]
```diff
@@ -329,6 +340,23 @@
...
+  "For each author, creates a page with filtered posts."
+  [{:keys [blog-prefix author-root-uri author] :as params} posts]
+  (println (blue "compiling authors"))
+  (create-folder (path "/" blog-prefix  author-root-uri))
+  ;; if the post author is empty defaults to config's :author
+  (doseq [{:keys [author posts]} (group-for-author posts author)]
+    (let [uri (path "/" blog-prefix author-root-uri (str author ".html"))]
+      (println "\t-->" (cyan uri))
+      (create-file uri
+                   (render-file "/html/author.html"
```
Comment (tomasz-biernacki) [inline src/cryogen_core/compiler.clj]: `/html/author.html` then needs to be added to the cryogen templates.

**Comment by lacarmen** [comment]
New version is on clojars :)

If you want to update the docs then that'd be great, otherwise I'll open an issue for myself.

**Comment by tomasz-biernacki** [comment]
I will update it

**Pull request merged**